queries.sql

```diff
@@ -0,0 +1,10 @@
+/*Queries that provide answers to the questions from all projects.*/
+
+SELECT * FROM animals WHERE name LIE '%mon';
+SELECT name FROM animals WHERE date_of_birth  BETWEEN '20160101' AND '20190101';
```

Comment:
- [ ] The data `20160101` is not an accurate date. Please update it to `2016/01/01`
Don't forget to end your check at `'2019/31/12` this will ensure that all animals born inside 2019 will be fetched.

Reply:
Thank You for the review :pray: